packages/fluentui/react-northstar/src/components/Form/Form.tsx

```diff
@@ -24,17 +24,6 @@ import {
   useAccessibility,
 } from '@fluentui/react-bindings';
 
-import { FormLabel } from './FormLabel';
```

Comment:
This doesn't need to be a breaking change immediately. Can we support both `FormLabel` and `Form.Label` for a version ?

Reply:
i.e. separate PRs to add and remove the Form components. 

Then we could release a patch version supporting both and help TMP adopt

Reply:
I think it's really the best to remove it now, because I saw of all the Form.Xxx, only Form.Field is used in TMP. The other way would be keep it for 53 and add a lint rule in TMP. We could talk tomorrow in open hour wdyt?

Reply:
Also, respect to the review speed O_o I'm sooo looking forward to meeting you in person, so I know you're real :D

Reply:
The current version supports both, I guess we can resolve this?